src/java.base/share/classes/java/time/Instant.java

```diff
@@ -1272,8 +1272,11 @@ public long toEpochMilli() {
      * It is "consistent with equals", as defined by {@link Comparable}.
      *
      * @param otherInstant  the other instant to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the {@code otherInstant} is before,
+     *          zero if they are equal, greater than zero if the {@code otherInstant} is after
```

Comment:
```suggestion
     * @return the comparator value - less than zero if {@code otherInstant} is before this instant,
     *          zero if they are equal, greater than zero if {@code otherInstant} is after this instant
```

Saying `the {@code otherInstant}` doesn't read correctly in text.

`the comparator value` needs punctuation after it in order to make sense